tools/dm/skip-replace-sqls.md

```diff
@@ -0,0 +1,530 @@
++ `sql-pattern`: 
+    - same with `--sql-pattern` of `sql-skip`
++ `sharding`: 
+    - same with `--sharding` of `sql-skip`
++ `task-name`: 
+    - same with `task-name` of `sql-skip`
++ `SQLs`: 
+    - Non-flag parameter, string, required 
+    - `SQLs` specifies the new SQL statements that are going to replace the original binlog event. You should separate multiple SQL statements with `;`, for example, ``` ALTER TABLE shard_db.shard_table drop index idx_c2;ALTER TABLE shard_db.shard_table DROP COLUMN c2; ```. 
+
+### Usage Example
+
+#### Passively skip after the replication gets interrupted
+
+##### Business scenario
+
+Assume that we need to replicate the upstream table `db1.tbl1` to the downstream TiDB (not in the scenario of merging and replicating data from sharded tables). The initial table schema is:
+
+```sql
+mysql> SHOW CREATE TABLE db1.tbl1;
++-------+--------------------------------------------------+
+| Table | Create Table                                     |
++-------+--------------------------------------------------+
+| tbl1  | CREATE TABLE `tbl1` (
+  `c1` int(11) NOT NULL,
+  `c2` decimal(11,3) DEFAULT NULL,
+  PRIMARY KEY (`c1`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 |
++-------+--------------------------------------------------+
+```
+
+Now, the following DDL statement is executed in the upstream to alter the table schema (namely, alter DECIMAL(11, 3) of c2 into DECIMAL(10, 3)):
+
+```sql
+ALTER TABLE db1.tbl1 CHANGE c2 c2 DECIMAL (10, 3);
+```
+
+Because this DDL statement is not supported by TiDB, the replication task of DM will get interrupted and report the following error: 
+
+```bash
+exec sqls[[USE `db1`; ALTER TABLE `db1`.`tbl1` CHANGE COLUMN `c2` `c2` decimal(10,3);]] failed, 
+err:Error 1105: unsupported modify column length 10 is less than origin 11
+```
+
+Now, if you query the status of the task using `query-status`, you can see that `stage` has changed into `Paused` and there is some related error description information in `errors`.
+
+To obtain the details about the error, you should use `query-error`. For example, you can execute `query-error test` to get the position of the failed binlog event (`failedBinlogPosition`), which is `mysql-bin|000001.000003:34642`. 
+
+##### Passively skip the SQL statement
+
+Assume that it is acceptable in the actual production environment that this DDL statement is not executed in the downstream TiDB (namely, the original table schema is retained). Then you can use `sql-skip` to skip this DDL statement to resume the replication. The procedures are as follows:
+
+1. Use `query-error` to obtain the position of the failed binlog event.
+    - You can get the position from `failedBinlogPosition` returned by `query-error`.
+    - In this example, the position is `mysql-bin|000001.000003:34642`.
+2. Use `sql-skip` to preset a skip operation that is to be executed when DM replicates this binlog event to the downstream after using `resume-task`.
+    ```bash
```

Comment:
Add a blank line before the code block (to make it looks better and avoid possible display error). Please also add a blank for other similar positions in this PR.